appserver/security/webservices.security/src/main/java/com/sun/enterprise/security/webservices/ClientPipeCreator.java

```diff
@@ -37,7 +37,7 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
-// Portions Copyright [2018] [Payara Foundation and/or its affiliates]
+// Portions Copyright [2018, 2022] [Payara Foundation and/or its affiliates]
```

Comment:
Usual format is with a hyphen, not commas
```suggestion
// Portions Copyright [2018-2022] [Payara Foundation and/or its affiliates]
```